src/cmds/import:data.ts

```diff
@@ -88,11 +88,33 @@ export async function handler(argv: {
         : `⚠ No import ${entityName[source]}(s) data generated!`;
 
     console.log(targetsMessage);
+
+    return { filename: res.fileName };
   } catch (e) {
     const errorMessage = `ERROR! Failed to generate data. Try running with \`DEBUG=snyk* <command> for more info\`.\nERROR: ${e}`;
     debug('Failed to generate data.\n' + e);
 
     console.error(errorMessage);
+    throw new Error(errorMessage);
```

Comment:
same as the other PR, please make the returns the same and we agreed to always print output outside of these functions in the `handler`